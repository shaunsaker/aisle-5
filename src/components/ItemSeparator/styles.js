import { StyleSheet } from 'react-native';

import styleConstants from '../../styleConstants';

const styles = StyleSheet.create({
  container: {
    alignSelf: 'stretch',
    height: 1,
    backgroundColor: styleConstants.colors.dividerColor,
  },
});

export default styles;
