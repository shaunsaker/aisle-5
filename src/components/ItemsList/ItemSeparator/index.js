import React from 'react';
import { View } from 'react-native';

import styles from './styles';

const propTypes = {};

const defaultProps = {};

const ItemSeparator = () => {
  return <View style={styles.container} />;
};

ItemSeparator.propTypes = propTypes;
ItemSeparator.defaultProps = defaultProps;

export default ItemSeparator;
