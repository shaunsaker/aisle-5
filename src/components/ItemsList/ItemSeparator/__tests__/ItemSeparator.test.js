import React from 'react';
import renderer from 'react-test-renderer';

import ItemSeparator from '..';

describe('ItemSeparator', () => {
  describe('renders', () => {
    it('renders with minimum required props', () => {
      const component = renderer.create(<ItemSeparator />);

      expect(component).toMatchSnapshot();
    });
  });
});
