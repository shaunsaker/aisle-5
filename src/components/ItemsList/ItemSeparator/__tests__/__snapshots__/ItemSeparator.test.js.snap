// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ItemSeparator renders renders with minimum required props 1`] = `
<View
  style={
    Object {
      "alignSelf": "stretch",
      "backgroundColor": "#FFFFFF",
      "height": 1,
    }
  }
/>
`;
